lawnchair/src/app/lawnchair/ui/preferences/components/colorpreference/ColorContrast Warning.kt

```diff
@@ -0,0 +1,63 @@
+package app.lawnchair.ui.preferences.components.colorpreference
```

Comment:
Looks like there's a space in the file name

Reply:
That's weird. Fixed it now.